create_aio_app/template/{{cookiecutter.project_name}}/docs/source/index.rst

```diff
@@ -55,3 +55,10 @@ Indices and tables
 * :ref:`genindex`
 * :ref:`modindex`
 * :ref:`search`
+
+.. This part only for the main site https://create-aio-app.readthedocs.io/
```

Comment:
Better do this through epilogue replacement...

Reply:
are u talking about that?
```
.. |reST| replace:: reStructuredText

Yes, |reST| is a long word, so I can't blame anyone for wanting to
abbreviate it.

```

Reply:
Yep. This way add whatever you want only for our RTD but don't ship it to users in templates.

Example: https://github.com/cherrypy/cherrypy/blob/master/docs/conf.py#L66-L70

Reply:
thx